public/app/features/alerting/unified/components/receivers/form/CloudReceiverForm.tsx

```diff
@@ -54,12 +54,16 @@ export const CloudReceiverForm = ({ contactPoint, alertManagerSourceName, readOn
 
   const onSubmit = async (values: ReceiverFormValues<CloudChannelValues>) => {
     const newReceiver = formValuesToCloudReceiver(values, defaultChannelValues);
-    if (editMode) {
-      await updateContactPoint({ contactPoint: newReceiver, originalName: contactPoint!.name });
-    } else {
-      await createContactPoint({ contactPoint: newReceiver });
+    try {
```

Comment:
Wrapping in a try/catch as otherwise the tests would fail with unhandled rejections